### PR TITLE
[products] Add latest products

### PR DESCRIPTION
**Pull request by alessio-palumbo**

### CODE REVIEW

File: products.json [hunk 1]
```diff
@@ -1699,9 +1699,643 @@
...
+          "temperature_range": [
+            1500,
+            9000
+          ]
+        },
+        "upgrades": []
+      },
+      {
+        "pid": 146,
+        "name": "LIFX Ceiling Intl",
```
Comment (stonehouse) [inline products.json]: nit: This is capitalised later, probably want to stick to `Intl` to be consistent as it's not an acronym

**Review by stonehouse — APPROVED** [review]

File: products.json (hunk 1 above)
Comment (alessio-palumbo) [reply products.json]: Good spotting, will fix

File: products.json [hunk 2]
```diff
@@ -1699,9 +1699,643 @@
...
+          "temperature_range": [
+            1500,
+            9000
+          ]
+        },
+        "upgrades": []
+      },
+      {
+        "pid": 147,
+        "name": "LIFX Round Spot US",
```
Comment (redding1) [inline products.json]: LCM3 version, can remove

File: products.json [hunk 3]
```diff
@@ -1699,9 +1699,643 @@
...
+          "temperature_range": [
+            1500,
+            9000
+          ]
+        },
+        "upgrades": []
+      },
+      {
+        "pid": 148,
+        "name": "LIFX Square Spot US",
```
Comment (redding1) [inline products.json]: LCM3 version, can remove

File: products.json [hunk 4]
```diff
@@ -1699,9 +1699,643 @@
...
+          "temperature_range": [
+            1500,
+            9000
+          ]
+        },
+        "upgrades": []
+      },
+      {
+        "pid": 149,
+        "name": "LIFX Round Path US",
```
Comment (redding1) [inline products.json]: LCM3 version, can remove

File: products.json [hunk 5]
```diff
@@ -1699,9 +1699,643 @@
...
+            1500,
+            9000
+          ]
+        },
+        "upgrades": []
+      },
+      {
+        "pid": 150,
+        "name": "LIFX Square Path US",
+        "features": {
```
Comment (redding1) [inline products.json]: LCM3 version, can remove

File: products.json [hunk 6]
```diff
@@ -1699,9 +1699,643 @@
...
+          "multizone": false,
+          "temperature_range": [
+            1500,
+            9000
+          ]
+        },
+        "upgrades": []
+      },
+      {
+        "pid": 151,
```
Comment (redding1) [inline products.json]: LCM3 version, can remove if needed.

File: products.json [hunk 7]
```diff
@@ -1699,9 +1699,643 @@
...
+            1500,
+            9000
+          ],
+          "extended_multizone": true
+        },
+        "upgrades": []
+      },
+      {
+        "pid": 152,
+        "name": "LIFX Outdoor Neon INTL",
```
Comment (redding1) [inline products.json]: LCM3 version, can remove if needed.

File: products.json [hunk 8]
```diff
@@ -1699,9 +1699,643 @@
...
+            1500,
+            9000
+          ],
+          "extended_multizone": true
+        },
+        "upgrades": []
+      },
+      {
+        "pid": 153,
+        "name": "LIFX PAR38 US",
```
Comment (redding1) [inline products.json]: LCM3 version, can remove if needed.

File: products.json [hunk 9]
```diff
@@ -1699,9 +1699,643 @@
...
+          "temperature_range": [
+            1500,
+            9000
+          ]
+        },
+        "upgrades": []
+      },
+      {
+        "pid": 154,
+        "name": "LIFX PAR38 INTL",
```
Comment (redding1) [inline products.json]: LCM3 version, can remove if needed.

File: products.json [hunk 10]
```diff
@@ -1699,9 +1699,643 @@
...
+          "temperature_range": [
+            1500,
+            9000
+          ]
+        },
+        "upgrades": []
+      },
+      {
+        "pid": 155,
+        "name": "LIFX A21 1600lm US",
```
Comment (redding1) [inline products.json]: LCM3 version, can remove if needed.

File: products.json [hunk 11]
```diff
@@ -1699,9 +1699,643 @@
...
+          "multizone": false,
+          "temperature_range": [
+            1500,
+            9000
+          ]
+        },
+        "upgrades": []
+      },
+      {
+        "pid": 156,
```
Comment (redding1) [inline products.json]: LCM3 version, can remove if needed.

File: products.json [hunk 12]
```diff
@@ -1699,9 +1699,449 @@
...
+          "temperature_range": [
+            1500,
+            9000
+          ]
+        },
+        "upgrades": []
+      },
+      {
+        "pid": 172,
+        "name": "LIFX Square Spot US",
```
Comment (redding1) [inline products.json]: Can remove this item. Not produced. (Square Spot)

File: products.json [hunk 13]
```diff
@@ -1699,9 +1699,449 @@
...
+            1500,
+            9000
+          ],
+          "extended_multizone": true
+        },
+        "upgrades": []
+      },
+      {
+        "pid": 205,
+        "name": "LIFX Outdoor Neon US",
```
Comment (redding1) [inline products.json]: Check I think this should be LIFX Indoor Neon US
#define LIFX_PRODUCT_ID_LCM4_INDOOR_NEON_SKIPPY_US      (0x00CD)   // SSID: LIFX Neon 123456 (Multi Zone Indoor Neon, US)

File: products.json [hunk 14]
```diff
@@ -1699,9 +1699,449 @@
...
+            1500,
+            9000
+          ],
+          "extended_multizone": true
+        },
+        "upgrades": []
+      },
+      {
+        "pid": 206,
+        "name": "LIFX Outdoor Neon Intl",
```
Comment (redding1) [inline products.json]: Update to indoor
#define LIFX_PRODUCT_ID_LCM4_INDOOR_NEON_SKIPPY_INTL    (0x00CE)   // SSID: LIFX Neon 123456 (Multi Zone Indoor Neon, INTL)

**Review by redding1 — APPROVED** [review]

**Pull request merged**